sys/net/network_layer/ng_ipv6/ng_ipv6.c

```diff
@@ -297,13 +297,12 @@ static void _send_multicast(kernel_pid_t iface, ng_pktsnip_t *pkt,
                             bool prep_hdr)
 {
     ng_pktsnip_t *netif;
+    kernel_pid_t *ifs;
 
-#if NG_NETIF_NUMOF > 1
-    /* netif header not present: send over all interfaces */
     if (iface == KERNEL_PID_UNDEF) {
         size_t ifnum;
```

Comment:
This needs to be moved out of this block, otherwise l320 will yield an error on NG_NETIF_NUMOF > 1.